frontend/__tests__/components/catalog.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { create } from 'react-test-renderer';
```

Comment:
Why are we using snapshot tests? We don't use them anywhere else because Enzyme is sufficient for testing unit testing components.

Reply:
I was able to replace several Enzyme tests with a single snapshot test.  Why test for individual pieces/attributes when we can compare a snapshot?

Reply:
While snapshot tests ensure consistency, they don't provide the same level of documentation for how the components should behave as Enzyme assertions.